doc/TSCC.rst

```diff
@@ -56,53 +54,53 @@ Submitting jobs
+SLURM files are bash script files with SLURM specific comments at the top.
 Example:
 
 .. code-block:: bash
 
-  #!/bin/bash
-  #PBS -q <queue>
-  #PBS -N <job_title>
-  #PBS -l nodes=1:ppn=1
-  #PBS -l walltime=<hours>:00:00
-  #PBS -o <log_dir>
-  #PBS -e <log_dir>
-  #PBS -V
-  #PBS -M <your email>
-  #PBS -m a
+  #!/usr/bin/env bash
+  #SBATCH --export ALL
+  #SBATCH --partition <partition>
+  #SBATCH --account ddp268
+  #SBATCH --qos <partition>
+  #SBATCH --job-name <job_title>
+  #SBATCH --nodes 1
+  #SBATCH --ntasks 1
+  #SBATCH --cpus-per-task 2
+  #SBATCH --time <hours>:00:00
+  #SBATCH --output slurm-%j.out-%N
+  #SBATCH --output slurm-%j.err-%N             # Optional, for separating standard error
   
   # ... do something ... 
 
-Google PBS to look up more information about these flags. Note that the PBS implementation on TSCC is likely to be slightly
-different than the implementation elsewhere, so online information won't be 100% accurate. In terms of naming conventions:
-tscc uses the job scheduler called `Torque`, `qsub` is the name of the command to submit a job to `Torque` and `PBS` is the 
-file format that `qsub` expects.
+Google "SLURM" to look up more information about these flags. In terms of naming conventions:
+tscc uses the job scheduler called SLURM and `sbatch` is the name of the command to submit a job to `SLURM`.
 
-The general workflow is to submit many jobs using the same PBS file, each with slightly different environment variable inputs
+The general workflow is to submit many jobs using the same SLURM file, each with slightly different environment variable inputs
 telling them to work on different input files. See below.
 
 Notes:
 
-* As above, :code:`qsub`'s user interface sucks. If you mistype the file it may hang forever without telling you. If successful
-  qsub should immediately return having printed out the name of your job, e.g. :code:`22409804.tscc-mgr7.local` telling you 
-  that's what happened. 
-* Aside from the first shebang line, PBS will stop looking for settings after the first line that does not start with :code:`#PBS`.
+* Aside from the first shebang line, SLURM will stop looking for settings after the first line that does not start with :code:`#SBATCH`.
   This includes blank lines and lines with comments.
+* The value for :code:`--account` is specific to our lab. If you aren't in our lab, you can use :code:`sacctmgr show assoc user=$USER format=account` to determine your lab's account.
 * Your job will be allocated memory in proportion to the number of processors your request. If you request 1 core and are assigned a node with 128GB
   of memory and 28 cores (the smaller of the two architectures of hotel nodes), your process will be allocated 128GB/28 = 4.57GB of memory and will
   be killed it if it exceeds that limit. If you want more memory, request more processors per node accordingly. (e.g. :code:`ppn=4`)
```

Comment:
For low memory error, you can directly use --memory <XXG> to request for memory. I usually use --cpus-per-task to run jobs parallel.

Reply:
sounds good. [This](https://stackoverflow.com/a/51141287/16815703) was the best explanation I could fine online, it seems like using `--ntasks 1 --cpus-per-task 4` will ensure that all four cpus are on the same node and are accessible by the same process. So it seems like that is the best way to go. In any case, it might be good to include a short explanation of all of this in addition to mentioning the recommended way.

Regarding your comment about `--memory`, are you suggesting that we can simply delete this entire section about allocating memory in proportion to the number of processors?

Reply:
I'm gonna close this thread in favor of https://github.com/gymrek-lab/gymreklab.github.io/pull/59/files#r1559849985